apps/retrospective/services/sprint_sync.go

```diff
@@ -295,6 +295,7 @@ func (service SprintService) AssignPoints(sprintID string) (err error) {
 		Not("sprint_tasks.id in (?)", sprintTaskToSkipPointsAllocation).
 		Scopes(retroModels.NotDeletedSprint).
 		Where("tasks.retrospective_id = ?", sprint.RetrospectiveID).
+		Not("tasks.done_at IS NULL").
```

Comment:
replace this with 
`where done_at is not null`

Reply:
It is already with a `Not`!!

Reply:
Agreed, but I'm not confident/comfortable with gorm's `Not`. 
`Not('column_name','value')` is okay but `Not("tasks.done_at IS NULL")` feels weird.
Can you share the actual query which is created when we write this statement

Reply:
Done